scripts/anvil_tools/add_or_remove_user_from_project.py

```diff
@@ -0,0 +1,111 @@
+import argparse
+from pandas import *
+import requests
+import urllib.parse
+from oauth2client.client import GoogleCredentials
+
+# function to get authorization bearer token for requests
+def get_access_token():
+    """Get access token."""
+
+    scopes = ["https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/userinfo.email"]
+    credentials = GoogleCredentials.get_application_default()
+    credentials = credentials.create_scoped(scopes)
+    return credentials.get_access_token().access_token
+
+
+def add_or_remove_user_from_project(project, email_list, add_action, verbose):
+    """Adding or Removing list of users' emails to the billing project"""
+    if verbose:
+        print(email_list)
+
+    # Get access token and and add to headers for requests.
+    # -H  "accept: */*" -H  "Authorization: Bearer [token] -H "Content-Type: application/json"
+    headers = {"Authorization": "Bearer " + get_access_token(), "accept": "*/*", "Content-Type": "application/json"}
+    
+    # Stopping running code at the 6th error to avoid multiple failures
+    error_count = 0
+
+    # Keeping track of emails added
+    emails_added = 0
+
+    for email in email_list:
+        # URL Encoding the email
+        url_encode_email = urllib.parse.quote(email)
+        
+        # Library/putLibraryMetadata
+        uri = f"https://rawls.dsde-prod.broadinstitute.org/api/billing/v2/{project}/members/user/{url_encode_email}"
+        
+        # capture response from API
+        if add_action:
+            response = requests.put(uri, headers=headers)
+            action="add"
```

Comment:
style nit: here and in lots of other places, please add spaces before and after `=` or `+=`

Reply:
Sure, I will add it now